com.playeveryware.eos/Runtime/Core/Common/SetOfNamed.cs

```diff
@@ -184,6 +205,24 @@ public bool Contains(T item)
 
             return false;
         }
+
+        public new bool Remove(Named<T> item)
```

Comment:
Why did you opt to use `new` here instead of `override`? I'm guessing that this is from inheriting from List.